src/agentic/cli.py

```diff
@@ -283,5 +304,281 @@ def run(args: List[str]):
     os.execvp("sh", ["sh", "-c", " ".join(args)])
 
 
+@app.command()
```

Comment:
Please move most of this code to a rag_helper or cli_rag file. Let's not just pack everything into this one file.